pkgs/development/haskell-modules/configuration-ghc-8.6.x.nix

```diff
@@ -45,7 +45,6 @@ self: super: {
   unordered-containers = dontCheck super.unordered-containers;
 
   # Test suite does not compile.
-  cereal = dontCheck super.cereal;
```

Comment:
This change is unrelated to purescript. It would be better to submit it in a separate PR, because this change is totally uncontroversial and it can be merged very quickly.